private/bufpkg/buflock/buflock.go

```diff
@@ -100,6 +108,7 @@ func ExternalConfigDependencyV1ForDependency(dep Dependency) ExternalConfigDepen
 		Owner:      dep.Owner,
 		Repository: dep.Repository,
 		Commit:     dep.Commit,
+		Digest:     dep.Digest,
```

Comment:
No cluster should be sending back any digest now, since it was disabled, and now in the new branch it's using the new field that stores the manifest digest. Although doing the check doesn't do any harm, I'd add it.

Reply:
I'll leave it out so we have less business logic in the CLI.